sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -931,9 +931,9 @@ Table {counter: tableNumber}. Additional member functions of the `sycl::handler`
 void handler::ext_oneapi_graph(command_graph<graph_state::executable>& graph)
 ----
 
-|Invokes the execution of a graph. Support for invoking an executable graph,
-before a previous execution of the same graph has been completed is backend
-specific. The runtime may throw an error.
+|Invokes the execution of a graph. Only one instance of `graph` may be executing,
```

Comment:
The name of the parameter in
```
void handler::ext_oneapi_graph(command_graph<graph_state::executable>& graph)
```
Is `graph` rather than `command_graph`, so prefer to keep it as-is